src/dropdown.ts

```diff
@@ -690,81 +682,93 @@ export default class Dropdown extends LitElement implements FormControl {
             ${ref(this.#dropdownElementRef)}
           >
             <span class="selected-option-labels" id="selected-option-labels">
-              ${this.selectedOptions.map(
-                ({ label }) =>
-                  html`<span data-test="selected-option-label">
+              ${this.selectedAndEnabledOptions
+                .filter((option) => {
+                  return this.multiple
+                    ? true
+                    : option === this.selectedAndEnabledOptions.at(-1);
+                })
+                .map(({ label }) => {
+                  return html`<span data-test="selected-option-label">
                     ${label},
-                  </span>`,
-              )}
+                  </span>`;
+                })}
             </span>
 
-            ${when(this.multiple && this.selectedOptions.length > 0, () => {
-              return html`<ul
-                aria-describedby="tag-overflow-text"
-                class="tags"
-                ${ref(this.#tagsElementRef)}
-              >
-                ${repeat(
-                  this.selectedOptions,
-                  ({ id }) => id,
-                  ({ id, editable, label, value }, index) => {
-                    return html`<li
-                      class=${classMap({
-                        'tag-container': true,
-                        hidden: index > this.tagOverflowLimit - 1,
-                      })}
-                      data-test="tag-container"
-                      data-test-hidden=${index > this.tagOverflowLimit - 1}
-                    >
-                      <glide-core-tag
-                        data-test="tag"
-                        data-id=${id}
-                        label=${ifDefined(label)}
-                        removable
-                        ?disabled=${this.disabled || this.readonly}
-                        ?private-editable=${editable}
-                        @edit=${this.#onTagEdit}
-                        @remove=${this.#onTagRemove.bind(this, id)}
+            ${when(
```

Comment:
Nothing has changed markup-wise except I'm now using `this.selectedAndEnabledOptions` instead of `this.selectedOptions`.